Sources/Utility/Git.swift

```diff
@@ -51,4 +51,18 @@ public class Git {
     public class var tool: String {
         return getenv("SWIFT_GIT") ?? "git"
     }
+
+    public class var version: String! {
+        return try? popen([Git.tool, "version"])
+    }
+
+    public class var majorVersionNumber: Int? {
+        let prefix = "git version"
+        var version = self.version
+        if version.hasPrefix(prefix) {
+            let prefixRange = version.startIndex...version.startIndex.advanced(by: prefix.characters.count)
+            version.removeSubrange(prefixRange)
```

Comment:
I can do this, but in that case there are to many expressions in 1 line, I think
```swift
version = String(version.characters[version.characters.startIndex.advanced(by: prefix.characters.count)..<version.characters.endIndex])
```